assets/css/wishlist.css

```diff
@@ -0,0 +1,107 @@
+@import "/assets/css/nav.css";
+@import "/assets/css/reset.css";
+
+.container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-gap: 1.5rem;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: repeat(3, 45rem);
+  padding-right: 7rem;
+  padding-left: 7rem;
+}
+
+.show-result-1 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 3rem;
+  padding-bottom: 3rem;
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+.card-img {
+  height: 30rem;
+  width: 100%;
+  background-color: var(--white);
+}
+
+.card-container {
+  width: 100%;
+  position: relative;
+}
+
+.card-1 {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: var(--black-shade) 0px 1px 4px;
+  background-color: var(--white);
+}
+
+.card-content {
+  padding-top: 0.6rem;
+  font-size: 1.2rem;
+  font-weight: 300;
+}
+
+.card-pricing {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.rupee-icons {
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.card-price {
+  font-size: 1.4rem;
+  padding-top: 0.5rem;
```

Comment:
It's a known fact in NEOG now, to use pixels below 1rem. Surely could skip that if you have a use case.

Reply:
Yes, I'll update.